metrics/web/lib/common/presentation/page_title/widgets/metrics_page_title.dart

```diff
@@ -55,11 +55,16 @@ class MetricsPageTitle extends StatelessWidget {
     );
   }
 
-  /// Navigates to the [DefaultRoutes.dashboard] page.
-  void _navigateHome(BuildContext context) {
+  /// Navigates to the previous page if [NavigationNotifier.canPop], 
+  /// otherwise navigates to the [DefaultRoutes.dashboard] page.
```

Comment:
🏷️ 
```suggestion
  /// Navigates to the previous page, if [NavigationNotifier.canPop].
  /// Otherwise, navigates to the [DefaultRoutes.dashboard] page.
```